code/modules/clothing/suits/ego_gear/he.dm

```diff
@@ -52,7 +52,7 @@
 	I’d be lying in my grave."
 	icon_state = "wings"
 	//Just Kinda meh. A lot of WAWs do black at the time of writing so
-	armor = list(RED_DAMAGE = 10, WHITE_DAMAGE = 40, BLACK_DAMAGE = -20, PALE_DAMAGE = 20)
+	armor = list(RED_DAMAGE = 30, WHITE_DAMAGE = 50, BLACK_DAMAGE = -10, PALE_DAMAGE = 0)
```

Comment:
HE armor probably shouldn't go to 50 or above.

Reply:
Egor both Grinder and Harvest both go up to 50. I dont know what system you're trying to get me to use because I used yours